java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -716,16 +847,13 @@ public ArrowBuf getBytes(int index, OutputStream out, int length) throws IOExcep
   }
 
   @Override
-  protected int _getUnsignedMedium(int index) {
```

Comment:
@siddharthteotia I added all methods back. This is the only exception. It's implemented in `getUnsignedMedium` now.